Projects/hCore/Sources/ScrollView+Inspect.swift

```diff
@@ -65,6 +65,83 @@ struct ViewIntrospector<ViewType: UIView>: UIViewRepresentable {
     }
 }
 
+class IntrospectionViewController: UIViewController {
```

Comment:
Remove if you didn't end up using this maybe?

Reply:
🆗